module/skill/Walk/data/config/frankie/Walk.yaml

```diff
@@ -41,3 +41,8 @@ arms:
   left_shoulder_roll: 0.35
   right_elbow: -0.7
   left_elbow: -0.7
+
+  kick:
+  # Step kick parameters
+  kick_velocity_x: 0.5 # Forward velocity during kick (m/s)
+  kick_velocity_y: 0.05 # Lateral velocity during kick (m/s)
```

Comment:
Is this right that Frankie uses the same values as in webots, rather than the default walk? Also indentation. I assume it should be the same as the default, so I'd recommend just removing this.